src/Components/UserContext/UserContext.js

```diff
@@ -0,0 +1,5 @@
+import { createContext } from 'react';
+
+const UserContext = createContext();
+
+export default UserContext;
```

Comment:
This should be a `.ts` file and you can use a named export (i.e. `export const UserContext...`)